server/tests/connectors/test_bitbucket_resolve_commit.py

```
@@ -0,0 +1,50 @@
+"""Bitbucket /src/ cannot take slashy branch names — _resolve_commit must hash them."""
+from unittest.mock import MagicMock
+
+from connectors.bitbucket_connector.api_client import BitbucketAPIClient
+
+
+def _client_with_gets(responses):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_client_with_gets`

(ANN202)